### PR TITLE
Jaisai new branch

### PR DESCRIPTION
**Pull request by Jaisai**

### CODE REVIEW

**Comment by Jaisai** [comment]
Hello,
I am Jaswanth Kumar, a new intern in the group. Please review my account and let me know for any errors.
Regards,
Jaswanth Kumar.

File: Jaisai.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Jaswanth Kumar Jonnalagadda, Maryville,MO/CST
+
+I am a passionate and committed Software developer with one year of work experience in Tata Consultancy services, INDIA (Multi National IT Service Company).
+
+• Posses good knowledge over MYSQL and MongoDB usage.
```
Comment (mappuji) [inline Jaisai.md]: There is a better way to create list you can see [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists).

Comment (dogi) [inline Jaisai.md]: thanks @mappuji

**Comment by waryhermit** [comment]
@Jaisai If you do not make the changes requested in 5 days this pull request will be closed. If you come back after that we can reopen it.

**Pull request closed without merging**